packages/app/src/cli/models/extensions/schemas.ts

```diff
@@ -21,6 +21,7 @@ export const TypeSchema = zod.object({
 export const NewExtensionPointSchema = zod.object({
   target: zod.string(),
   module: zod.string(),
+  label: zod.string().optional(),
```

Comment:
We might want to do validation to make this required for the Action extension schema in Core later on